Makefile.codegen

```diff
@@ -57,8 +68,12 @@ generate-clean: clean ## Delete code generation tools
 	@rm -f $(GOPATH)/bin/gen-apidocs
 	@rm -f $(GOPATH)/bin/pegomock
 
+stash:
+	# Making sure repo has no outstanding changes
```

Comment:
This will make sure any pre-existing modifications gets stashed prior to verify-generation-complete